### ISSUE TITLE
ITC-3 doesn't respond with current values when a command fails

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_validator#297]
When a command fails, the command response must include the current values back, not the values that were requested.

In this example, am M0014 command to restart with status=True was send, but with an invalid security code. The TLC sends status=True back, which is incorrect, because the TLC doesn't actually restart.

```
2021-11-01T12:38:02.899Z 56186    KK+AG9998=001TC000 KK+AG9998=001TC000 335a  Received CommandResponse {"mType":"rSMsg","type":"CommandResponse","mId":"335a4c6f-39ce-4b45-a19f-90f24a2258f5","ntsOId":"","xNId":"","cId":"KK+AG9998=001TC000","cTS":"2021-11-01T12:36:10.771Z","rvs":[{"cCI":"M0004","age":"recent","n":"status","v":"True"},{"cCI":"M0004","age":"recent","n":"securityCode","v":""}]}
```

**Comment by SwarcoPalm** [comment]
Is this test covered by the validator?

**Comment by emiltin** [comment]
I don't think we currently have a tests that checks that current value are returrned when a commad fails. But I think it would be useful, and not hard to write. We can send a command request with invalid values to ensure it always fails.

**Comment by emiltin** [comment]
Open a new issue about adding a test for returning current values at #436.
